src/utils/use-users-handlers.js

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getUser } from '../redux/users/users.actions';
+
+const useUsersHandler = (id) => {
+  const dispatch = useDispatch();
+
+  const { user } = useSelector(({ Users }) => ({
+    user: Users.user
+  }));
+
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+
+  const [country, setCountry] = useState('');
+  const [city, setCity] = useState('');
+  const [adress, setAdress] = useState('');
+  const [postCode, setPostCode] = useState('');
+
+  const [ban, setBan] = useState('');
```

Comment:
Please, rename `ban` to the `isBanned`, it will increase the readability of the code

Reply:
renamed